pom.xml

```diff
@@ -1277,7 +1277,7 @@
               <exclude>**/.settings/**/*</exclude>
 
               <!-- Proto/grpc generated wrappers -->
-              <exclude>**/sdks/python/apache_beam/runners/api/*.py</exclude>
+              <exclude>**/apache_beam/runners/api/*_pb2*.py</exclude>
```

Comment:
Yes, there is a bit of a tangle here that we should clean up (in a future PR). Open for suggestions on where best to put these though.